src/main/java/hudson/plugins/octopusdeploy/OctopusDeployReleaseRecorder.java

```diff
@@ -393,7 +408,7 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
 
                     String deploymenturlSuffix = api.getDeploymentsApi().getPortalUrlForDeployment(fullProject.getId(), releaseVersion, fullEnvironment.getId(), tenantId);
```

Comment:
And this one's a deployment endpoint, which _has_ to have the associated release. Makes sense 👍